package.json

```diff
@@ -60,14 +59,15 @@
     "react-dropzone": "^14.2.2",
     "react-error-boundary": "^3.1.4",
     "react-shadow": "^19.0.3",
-    "spc-parser": "^0.5.2",
+    "spc-parser": "^0.6.0",
     "tinycolor2": "^1.4.2",
     "use-resize-observer": "^9.0.2",
     "wdf-parser": "^0.2.1"
   },
   "devDependencies": {
     "@babel/core": "^7.19.3",
     "@babel/eslint-parser": "^7.19.1",
+    "@emotion/react": "^11.10.4",
```

Comment:
This has to stay in `dependencies`

Reply:
done sorry about that i didn't pay attention